contrib/osx/keybindings.el

```diff
@@ -30,4 +30,6 @@
                       (interactive)
                       (call-interactively (key-binding "\C-x\C-s"))))
     (global-set-key (kbd "s-Z") 'undo-tree-redo)
-    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)))
+    (global-set-key (kbd "C-s-f") 'spacemacs/toggle-frame-fullscreen)
+    ;; Emacs sometimes registers C-s-f as this weird keycode
+    (global-set-key (kbd "<C-s-268632070>") 'spacemacs/toggle-frame-fullscreen)))
```

Comment:
Who says you have to memorise it? It's the exact equivalent of pressing C-s-f on my mac, and on many other peoples' as well (http://stackoverflow.com/questions/27206536/how-do-you-set-a-key-with-both-cmd-and-ctrl-in-emacs-app, https://github.com/preetpalS/decl.el, https://lists.gnu.org/archive/html/bug-gnu-emacs/2008-11/msg00011.html). You just have to set it and forget it.